Userland/Services/WindowServer/WindowFrame.cpp

```diff
@@ -383,12 +383,14 @@ void WindowFrame::paint(Gfx::Painter& painter, const Gfx::IntRect& rect)
     }
 }
 
-void WindowFrame::paint_completely(Gfx::Painter& painter) {
+void WindowFrame::paint_completely(Gfx::Painter& painter)
+{
```

Comment:
Tons of unrelated formatting changes in this commit. Please fix formatting in the earlier commit instead :)

Reply:
Is fixed now. :-)